src/components/InventoryV2/BatchDetailsModal.tsx

```diff
@@ -236,7 +235,7 @@ export default function BatchDetailsModal(props: BatchDetailsModalProps): JSX.El
             type='number'
             label={strings.GERMINATING_QUANTITY_REQUIRED}
             tooltipTitle={strings.TOOLTIP_GERMINATING_QUANTITY}
-            errorText={validateFields && !record.germinatingQuantity ? strings.REQUIRED_FIELD : ''}
+            errorText={validateFields && record?.germinatingQuantity === undefined ? strings.REQUIRED_FIELD : ''}
```

Comment:
seen this before I think..
thoughts on a util we can reuse?
isNumber = !isNaN(Number(val))